app/jobs/notification_sender_job.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+# notification job
+class NotificationSenderJob < ApplicationJob
+  queue_as :notifications
+
+  def perform(id, class_name)
```

Comment:
call only one method within `perform`, and move the private methods to another model class or reference service.

Reply:
done!